src/app/utils/index.ts

```diff
@@ -315,7 +315,11 @@ export function convertColumns(
   });
 
   try {
-    const convertedValues = convertColumn(type, columnValues as any);
+    const convertedValues = convertColumn(
+      type,
+      columnValues as any,
+      table.LocaleNumberFormat
```

Comment:
Nitpick: `LocaleNumberFormat` to `localeNumberFormat`, it's consistent with the other properties in `Table`

Reply:
That was unintentional, good catch!